### PR TITLE
Added Dsc Jobs

### PR DESCRIPTION
**Pull request by AhmedSabry957**
image
image

### CODE REVIEW

File: websites/D/Dscjobs.org/presence.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      window.location.search
+    ),
+    myParam: string = urlParams.get("term");
+    presenceData.details = "Searching:";
+    presenceData.state = myParam;
+  } else if (page === "/settings") {
+    presenceData.details = "Editing his Profile";
+  } else if (page === "/faq") {
+    presenceData.details = "Browsing:";
+    presenceData.state = "FaQ Page";
```
Comment (ririxidev) [inline websites/D/Dscjobs.org/presence.ts:31]: there is nothing like `FaQ`. There is only `FAQ`

File: websites/D/Dscjobs.org/presence.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  } else if (page.includes("/cv/")) {
+    if (page.includes("/rate")) {
+      presenceData.details = "Rating for:";
+      presenceData.state = document.querySelector("#box1 > div.vote_box > h1").textContent;
+      presenceData.buttons = [
+        {
+          url: document.URL,
+          label: "Rate Profile"
+        }
+      ];
```
Comment (ririxidev) [inline websites/D/Dscjobs.org/presence.ts:49]: Ehhh, kinda unnecessary

File: websites/D/Dscjobs.org/presence.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  },
+    page = document.location.pathname;
+
+  if (page === "/") {
+    presenceData.details = "Browsing:";
+    presenceData.state = "Main Page";
+  } else if (page === "/profile") {
+    const username: string = document.querySelector("body > div.profile_header > div > div > h1").textContent;
+    presenceData.details = "Watching Profile:";
+    presenceData.state = `${username}`;
```
Comment (ririxidev) [inline websites/D/Dscjobs.org/presence.ts:19]:
```suggestion
    presenceData.state = username;
```

**Review by ririxidev — CHANGES_REQUESTED** [review]
also add all subpages on site.

**Comment by AhmedSabry957** [comment]
fixed

**Comment by ririxidev** [comment]
If fixed why you close it? Lmfao.

**Comment by AhmedSabry957** [comment]
@ririxidev My friend will reopen it I have little exp in github, but I will try and reopen

**Comment by ririxidev** [comment]
I mean if you want someone else to make a pull request, go ahead but close this one.

**Comment by AhmedSabry957** [comment]
> I mean if you want someone else to make a pull request, go ahead but close this one.

try and see if everything is really fixed, and if so approve it

**Comment by AhmedSabry957** [comment]
so?

**Review by ririxidev — CHANGES_REQUESTED** [review]
Add all subpages that you can find on website and 
image
Also we are not reviewing 24/7 so please wait when we don't respond...

File: websites/D/Dscjobs.org/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Dsc Jobs",
```
Comment (Bas950) [inline websites/D/Dscjobs.org/dist/metadata.json:3]: Folder name and service name must be the same

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by AhmedSabry957** [comment]
fixed

**Comment by AhmedSabry957** [comment]
anything else?

**Comment by ririxidev** [comment]
image

**Comment by AhmedSabry957** [comment]
you mean I should remove?

**Review by AhmedSabry957 — COMMENTED** [review]
Fixed

**Comment by ririxidev** [comment]
It's not fixed. Man. I think you still don't understand me. 
It's not only about this button. I said to add all subpages that are available on this site to your presence. If you will still not understand, unfortunately I will just close this pull request.

File: websites/D/Dscjobs.org/dist/metadata.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://schemas.premid.app/metadata/1.3",
-    "service": "Dsc Jobs",
+    "service": "Dscjobs.org",
```
Comment (Bas950) [inline websites/D/Dscjobs.org/dist/metadata.json]: You should just change the folder name and service to "DscJobs" as that's their official name

**Comment by AhmedSabry957** [comment]
I will get someone expert do this

**Pull request closed without merging**